src/index.html

```diff
@@ -5,9 +5,93 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card-block" data-qa="card">
+      <a href="#" class="product">
+        <img
+       src="images/imac.jpeg"
+       class="product__img"
+       alt="Product"
+       >
```

Comment:
```suggestion
        <img
         src="images/imac.jpeg"
         class="product__img"
         alt="Product"
       >
```

If the attribute is in a new line, it should be indented two spaces more from the left edge than the opening tag